src/App.js

```diff
@@ -1,11 +1,26 @@
 import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import Navbar from './components/Navbar';
+import Mission from './components/Mission';
+import Rockets from './components/Rockets';
+import MyProfile from './components/MyProfile';
 
 function App() {
   return (
-    <div className="App">
-      <Navbar />
-    </div>
+    <Router>
+      <div className="App">
+        <div className="nav">
+          <Navbar />
+        </div>
```

Comment:
What do you think of creating the div that you did right here in the Navbar component wouldn't that be better since the main component contains a lot of code?